app/models/enumeration/EventType.scala

```diff
@@ -87,6 +87,8 @@ object EventType extends Enumerable.Implicits {
         stringBinder.unbind(key, value.toString)
     }
 
+  implicit val jsLiteral: JavascriptLiteral[EventType] = (value: EventType) => value.toString
```

Comment:
why do we need this implicit ?

Reply:
It was added to stop the event from being added to the URL in the page. The Address page has something similar.